cli/cmd/devtool/start.go

```diff
@@ -183,16 +188,30 @@ type servicesCfg struct {
 	deps    bool
 	runtime bool
 	ui      bool
+	minimal bool
 	only    []string
 	except  []string
 }
 
 func (s *servicesCfg) addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringSliceVar(&s.only, "only", []string{}, "Only start the listed services (options: admin, deps, runtime, ui)")
 	cmd.Flags().StringSliceVar(&s.except, "except", []string{}, "Start all except the listed services (options: admin, deps, runtime, ui)")
+	cmd.Flags().BoolVar(&s.minimal, "minimal", false, "Start minimal services: deps (postgres, redis), runtime, and admin")
 }
 
 func (s *servicesCfg) parse() error {
+	if s.minimal {
+		if len(s.only) > 0 || len(s.except) > 0 {
+			return errors.New("cannot use --minimal with --only or --except")
+		}
+		s.deps = true
+		s.runtime = true
+		s.admin = true
+		s.ui = false
+
+		return nil
+	}
```

Comment:
I don't think it should disable the UI always, that's really less than "minimal". If you want to run the UI separately, then you could just pass `--minimal --except ui`.